eanzalone/passion_project.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<header></header>
+<body>
+	<!-- Write 1 paragraph about a project I'm passionate about -->
+<p>I have so many ideas! There are three I'm especially passionate about, so I'll talk about all of them because one could potentially go nowhere, depending on legality. <br /> The legally dicey idea is a Terms of Service Sparknotes. No one wants to read those long, boring Terms of Service. But at the same time, you don't want to download Facebook Messanger only to find out you agreed to let Facebook see all sorts of information outside their apps. I want to create an app, or site, which highlights all the key factors you're agreeing to.<br />My second idea is a full access iPhone feature that blocks you from texting certain people. My friend and I came up with it while discussing texts you regret sending when you wake up. I figure, if Keyboard apps can have full access, why couldn't that be applied elseware.<br />My final idea is more of a user-input database about hair color. I've been dying my hair for years and have built up a large knowledge base of products. It would be wonderful to have an app where people could review products and talk about how they worked for them, with extra information like hair type. It wouldn't be a shopping app, but maybe I could add in links to products on Amazon.</p>
```

Comment:
I really like your first idea.  In the open-source software world we have some very commonly used licenses that many of us just re-use: MIT, GPL, LGPL, etc.